integration_tests/numerical_test.py

```diff
@@ -41,7 +44,8 @@ def build_keras_model(keras_module, num_classes):
             keras_module.layers.Conv2D(
                 64, kernel_size=(3, 3), activation="relu"
             ),
-            keras_module.layers.BatchNormalization(scale=False, center=True),
+            # TODO: Renable the following line.
```

Comment:
What is the issue? Numerical difference across backends?

Reply:
Yes, if I add this layer, TF and JAX tests would fail due to numerical differences.